tailwind.config.js

```diff
@@ -14,10 +14,10 @@ module.exports = {
       success: '#21cdac',
       gray: {
         light: '#e5e5e5',
-        lighter: '#fafafa'
+        lighter: '#fafafa',
       },
-      primary: '#21cdac',
-      secondary: '#28a7db'
+      primary: 'var(--primary-color)',
+      secondary: 'var(--secondary-color)',
```

Comment:
Let's set up `.prettierrc` to remove/prevent trailing commas. Unless you want to keep them?